### PR TITLE
Chore/rel 1.x prep

### LINKED ISSUE #116: Remove the `-multi` tag from cross-platform images
Why is there a need to append `-multi` to the tag? If the `latest` tag would be cross-platform:

* as a user you would not need to know about the `-multi` tag
* users would download the same amount of data from the image registry

### LINKED ISSUE #117: V1.x Release
Hey all,

Just to let you know we will be promoting to a V1.x release very shortly, 

It will do the following at the moment

- Promote to 1.x (long overdue 
- Drop the `-latest` suffix from images, and produce multi-arch images against the `latest` tag
  - This will allow arm64/arm users to natively use the `latest` flag without having to force a `linux/amd64` platform. #116 
- Revert to the previous version scheme
  - New changes were introduced and documented [here](https://github.com/pact-foundation/pact-ruby-cli#versioning)
  - The versioning scheme used the M.m.p from the Pact Cli gem, with an additional RELEASE number at the end (eg. 0.51.0.4)
    - pact-cli gem version number https://github.com/pact-foundation/pact-ruby-cli/blob/master/lib/pact/cli/version.rb

### PR DESCRIPTION
**Pull request by YOU54F**
fixes #117 
closes #116 

- Reverts unnecessarily complicated versioning scheme introduced in May
- Publishes multi-manifest images to latest flag, thus removing the need to append `-multi` to image tags.

### CODE REVIEW

**Comment by YOU54F** [comment]
Tested against my fork, albeit with the wrong image name (🤦🏾 😂 )

Screenshot 2023-08-30 at 17 20 07

Screenshot 2023-08-30 at 17 21 07

File: .github/workflows/update-and-prepare-release.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      run: script/workflows/update.sh
+      env:
+        RELEASED_GEM_NAME: ${{ github.event.client_payload.name }}
+        RELEASED_GEM_VERSION: ${{ github.event.client_payload.version }}
+
+    - name: Commit and push gem file changes
+      uses: EndBug/add-and-commit@v4
+      with:
+        add: Gemfile.lock
+        author_name: Beth Skurrie via Github Action
```
Comment (bethesque) [inline .github/workflows/update-and-prepare-release.yml:31]: Could use the github action user here. I probably didn't know about it when I hardcoded this.

File: .github/workflows/update-gems.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# TODO abort if there are no changes to the Gemfile.
```
Comment (bethesque) [inline .github/workflows/update-gems.yml:1]: I think this probably does abort due to line 23 of update.sh

File: .github/workflows/update-and-prepare-release.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# TODO abort if there are no changes to the Gemfile.
```
Comment (bethesque) [inline .github/workflows/update-and-prepare-release.yml:1]: What is this workflow used for? If it's copied from another repo where I wrote it, it's probably not necessary.

**Review by bethesque — APPROVED** [review]

File: .github/workflows/update-and-prepare-release.yml (hunk 3 above)
Comment (YOU54F) [reply .github/workflows/update-and-prepare-release.yml:1]: It came back from the original repo, but was actually never connected up to a script

Ie `script/workflows/prepare-release.sh` this path doesn't exist that its actually calling, so will pull this !

File: .github/workflows/update-and-prepare-release.yml (hunk 1 above)
Comment (YOU54F) [reply .github/workflows/update-and-prepare-release.yml:31]: yep makes sense to update it now, whilst we are here

File: .github/workflows/update-gems.yml (hunk 2 above)
Comment (YOU54F) [reply .github/workflows/update-gems.yml]: It will yeah, good spot

File: .github/workflows/update-and-prepare-release.yml (hunk 3 above)
Comment (YOU54F) [reply .github/workflows/update-and-prepare-release.yml]: image

**Pull request merged**